app/lib/r/analyze.R

```diff
@@ -0,0 +1,45 @@
+library(RPostgreSQL)
+
+# funkcia nacita ulozeny csv subor do framu
+readCSV = function(f){
+	csv <- read.csv(file=f, sep=";")
+	return(csv)
+}
+
+# funkcia vyberie ciselne stlpce a spravi nad nimi statisticke funkcie, ktore zoskupi do summary framu
+analyze = function(csv){
+	nums <- sapply(csv, is.numeric)
+	min <- apply(csv[nums], 2, min)
+	max <- apply(csv[nums], 2, max)
+	median <- apply(csv[nums], 2, median)
+	mean <- apply(csv[nums], 2, mean)
+
+	summary <- rbind(min, max, median, mean)
+	return(summary)
+}
+
+# funkcia vezme summary frame a vlozi jeho hodnoty do tabulky
+DBinsert = function(summary, dataset_id){
+	drv <- dbDriver("PostgreSQL")
+	con <- dbConnect(drv, dbname="Skola_development")
```

Comment:
Tento riadok asi nepobezi na produkcnej verzii ...Treba dat vediet chalanom ked to budu nasadzovat co tam treba doplnit